cfdtools/_cli.py

```diff
@@ -5,8 +5,6 @@
 
 # readers and writers - must be imported to update api format dict
 import cfdtools.ic3 as ic3  # .reader_legacy # needed to map readers
-import cfdtools.gmsh as gmsh
```

Comment:
I thought it was compulsory to add them in order to fill the dictionnary of readers, did you check it still works ? (for example, ic3write command from a cgns file)

Reply:
I have tried to change `test_1_cgns.py` with:
```
+import cfdtools.ic3.writerV3 as ic3writer
+import cfdtools.api as api
...
+    ic3write = ic3writer.writer(rmesh)
+    _builddir.mkdir(exist_ok=True)
+    outfile = api._files(_builddir / filename)
+    outfile.change_suffix('.ic3')
+    ic3write.write_data(outfile.filename)
+    Path(outfile.filename).unlink()
```
And it worked. So I think this is ok ?